test/module/irohad/ametsuchi/wsv_query_command_test.cpp

```diff
@@ -190,6 +187,19 @@ CREATE TABLE IF NOT EXISTS account_has_grantable_permissions (
       ASSERT_EQ(account.json_data, acc.value().json_data);
     }
 
+    /**
+     * @given inserted role, domain, account
+     * @when update account json data
+     * @then get account and check json data is the same
+     */
+    TEST_F(AccountTest, UpdateAccountWithJSONData) {
```

Comment:
I suggest adding a test that checks complex JSON data: arrays, nested objects, different types.

Reply:
Also, what about update and delete data?

Reply:
No delete currently needed